samples/client/petstore/rust/src/apis/get_pet_by_id_api.rs

```diff
@@ -0,0 +1,23 @@
+use hyper;
+use serde_json;
+use futures::{Future, Stream};
+
+use super::models;
+use super::Error;
+
+pub fn get_pet_by_id<C: hyper::client::Connect>(
+    prefix: &str,
+    cli: &hyper::client::Client<C>,
+    pet_id: i64,
+    ) -> Box<Future<Item = models::Pet, Error = Error>> {
```

Comment:
All this code is async-only. I don't think that it makes sense to have a matching sync version.